config/locales/en.notifier.yml

```diff
@@ -38,6 +38,8 @@ en:
           New transactions have moved into the billing workflow. The default review
           period is seven days, after which the charges will be invoiced/journaled.
 
+          For any billing-related questions, please email <%{email}>
```

Comment:
I think that facility.email is an optional attribute - can you ask the client how to handle the case where a facility has not specified an email address?